src-java/flowmonitoring-topology/flowmonitoring-storm-topology/src/main/java/org/openkilda/wfm/topology/flowmonitoring/service/ActionService.java

```diff
@@ -166,6 +193,14 @@ public void sendFlowSyncRequest(String flowId) {
                 log.info("Sending flow '{}' sync request.", flowId);
```

Comment:
method `saveFlowLatency` will send stats with just `flowid` (which is sub flow id in case of HA flow). It means that FlowStatsService wouldn't find common flow with ID if subFlow and log `Can't save latency for flow` for each HaSubFlow